app/views/users/index.html.erb

```diff
@@ -2,18 +2,19 @@
 <h1>List of all the users</h1>
 
 <div class="wrapper">
-  <h1>Users</h1>
+<h1>Users</h1>
   <ul>
     <% @users.each do |user| %>
       <li class="userNameWrapper">
-      <div><img alt="Profile P."/></div>
-      <div class="userName">
-      <%= link_to user_path(user), class: 'userNameAnchor' do %>
-        <h2><%= user.name %></h2>
-        <h6>Number of posts: <%= user.posts.count %></h6>
+          <div><img alt="Profile P."/></div>
+            <div class="userName">
+          <%= link_to user_path(user), class: 'userNameAnchor' do %>
+            <h2><%= user.name %></h2>
+            <h6>Number of posts: <%= user.posts.count %></h6>
```

Comment:
**OPTIONAL** - Good job done so far, but to help with the user experience for your project, it'd be better if you include a `Add new post` button that directs the user to the form for adding a new post, just as you did for the comments. Please fix this 😄

Reply:
okay i will do it, thank you so much, i really appreciate it